docs/compiler.adoc

```diff
@@ -5,8 +5,9 @@ https://github.com/intel/cpp-std-extensions/blob/main/include/stdx/compiler.hpp[
 provides macros for decorating declarations, which resolve either to keywords or
 to compiler-specific attributes:
 
+* https://clang.llvm.org/docs/AttributeReference.html#always-inline-force-inline[`ALWAYS_INLINE`]
 * https://en.cppreference.com/w/cpp/language/constinit[`CONSTINIT`]
 * https://en.cppreference.com/w/cpp/language/consteval[`CONSTEVAL`]
```

Comment:
suggestion: if you want these to be alphabetical then you should swap `CONSTINIT` and `CONSTEVAL`